README.md

```diff
@@ -15,6 +15,10 @@ Google Santa Tracker for Android
 * Interactive Android Wear watchfaces (with sound!)
 * Videos, animations and more.
 
+## In the 2017 version, new features include:
+
+* Corrected all problems
```

Comment:
:smile: I wish this was true!  But I don't think this is a useful changelog since we certainly did not fix all problems in the app.